### DISCUSSION TITLE
How to build?

### DISCUSSION THREAD

**Discussion by cacklinglaughing** [HttpRafa/modflared#38] [Ideas]
I've only ever used the original ForgeGradle so I don't know how to build this mod from source, specifically for Eclipse or VS Code for the Forge version 1.16.5.

**Comment by HttpRafa** [comment]
Modflared also uses gradle so just run `.\gradlew.bat build` in a CMD window on Windows. Or `./gradlew build` on Linux. The builded jar is in fabric/build/libs ... or forge/build/libs
